scripts/migrate_piwik_historical_data.py

```diff
@@ -0,0 +1,152 @@
+import MySQLdb
+from socket import inet_ntoa
+from binascii import b2a_hex
+import pprint
+from keen import KeenClient
+import urlparse
+
+def main():
+
+    keen_client = KeenClient(
+        project_id='56abc1c759949a6734c3f32b',
```

Comment:
This script won't be used for a while, so that id is just a project id I had for a test on Keen to test the script. It will be implemented correctly later, but now it simply exists as a placeholder until proper migration occurs.